geomstats/visualization/special_linear.py

```diff
@@ -0,0 +1,295 @@
+"""Visualization for Geometric Statistics."""
+
+from functools import partial
+
+import geomstats.backend as gs
+import matplotlib.animation as animation
+import matplotlib.pyplot as plt
+from geomstats.geometry.matrices import Matrices
+from geomstats.geometry.special_linear import SpecialLinear
+from mpl_toolkits.mplot3d import Axes3D  # NOQA
+from mpl_toolkits.mplot3d.art3d import Poly3DCollection
+
+SL2 = SpecialLinear(n=2)
+SL3 = SpecialLinear(n=3)
+
+AX_SCALE = 1.2
+
+
+class Squares:
+    """Class used to draw the unit square and transformations using SL2(R)."""
+
+    def __init__(self, num_points=10, points=None):
+        """Squares() constructor.
+
+        Construct an instance of the Squares() class.
+
+        Parameters
+        ----------
+        num_points : int
+            The number of points to use for the representation of the unit
+            square along one axis
+        points : array-like, shape=[..., 2, 2]
+            Points on the SL2(R) manifold of 2D matrices with a determinant of
+            1.
+        """
+        self.num_points = num_points
+        x = gs.linspace(-0.5, 0.5, num_points)
+        y = gs.linspace(-0.5, 0.5, num_points)
+        xcoord, ycoord = gs.meshgrid(x, y)
+        self.unit_X = gs.reshape(xcoord, (xcoord.size, 1))

```

Comment:
no uppercase letters in variable names in python (except for classes and constant variables) --> change to lowercase

(the variables at the top of your file, like SL2, are constant variables so that is fine)